docs/src/modules/java-protobuf/partials/effects-api.adoc

```diff
@@ -0,0 +1,11 @@
+
+[#_effects_api]
+== Understanding what an Effect is
+
+Each component defines a set of operations through its application programming interface (API). These operations are specific to the semantics of each component. For the JVM SDKs, these APIs take the shape of an Effect.
+
+An Effect is a description of what Kalix needs to do after an incoming command is handled. You can think of it as a set of instructions you are passing to Kalix. Kalix will process the instructions on your behalf.
```

Comment:
```suggestion
An Effect is a description of what Kalix needs to do to handle an incoming command. You can think of it as a set of instructions you are passing to Kalix. Kalix will process the instructions on your behalf.
```

Reply:
after command is handler, sounds strange, the effect is the instruction how the command should be handled

Reply:
I see your point, but it might be confusing. 

The command handler is the user defined method that can do many things, validation, call other components in case of Action, etc. It does handle the command and run business logic. 

The result of this handling is passed to Kalix for further processing. In a sense, the handling of a command is a shared responsibility where part of it is defined and executed by the user's code and part of it is handled by Kalix.

The choice of using the phrasing "after an incoming command is handled" is intentional because it emphasizes that after the user code is executed, something else will be executed on their behalf.